packages/sdk/src/store/persisted.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Safe IDB storage interface. These try..catch are useful because
+ * some browsers may block access to these APIs due to security policies
+ *
+ * Also, the stored value is lazy-loaded to avoid hydration mismatch
+ * between server/browser. When state is 'hydrated', the value in the heap
+ * is the same as the value in IDB
+ */
+ import { get, set } from "idb-keyval";
+ import { Store } from "./base";
+ 
+ const getIDB = async <T>(key: string) => {
+   try {
+     return await get<T>(key);
+   } catch (err) {
+     return;
```

Comment:
```suggestion
     throw err;
```

Reply:
it is better not throwing the error to make it work on all environments, like Node.JS or older browsers

Reply:
so why try catch?

Reply:
to make it work on older browsers. The idea is that the `get` will throw on older browers, but the application will keep working because of the try ... catch and the result will be stored on memory only